packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,122 @@
+import fs from "fs";
+import path from "path";
+import inquirer from "inquirer";
+import { run, RunOptions } from "npm-check-updates";
+import { installDependencies } from "../utils/installDependencies.js";
+import { Index } from "npm-check-updates/build/src/types/IndexType.js";
```

Comment:
You don't need to import this `Index` type

Reply:
The Index type comes in handy to represent any key value pair index, shall I create an interface by myself in the file instead?

Reply:
Removed.